src/applications/vaos/new-appointment/redux/selectors.js

```diff
@@ -242,6 +231,9 @@ export function selectSingleValidVALocation(state) {
 
   return validFacilities?.length === 1 && !!data.vaFacility;
 }
+export function hasSingleValidVALocation(state) {
+  return getNewAppointment(state)?.data?.isSingleVaFacility;
+}
```

Comment:
I've been trying to follow these recommendations. I think this is a good read for the team. https://redux.js.org/usage/deriving-data-selectors

 _"A selector function can have any name you want. However, [we recommend prefixing selector function names with the word select combined with a description of the value being selected](https://redux.js.org/style-guide/#name-selector-functions-as-selectthing)."_

Reply:
> I've been trying to follow these recommendations. I think this is a good read for the team. https://redux.js.org/usage/deriving-data-selectors
> 
> _"A selector function can have any name you want. However, [we recommend prefixing selector function names with the word select combined with a description of the value being selected](https://redux.js.org/style-guide/#name-selector-functions-as-selectthing)."_

Thanks @vbahinwillit , I'll rename the selector function to follow recommended naming convention.